### DISCUSSION TITLE
Time Varying Media Parameters

### DISCUSSION THREAD

**Discussion by shubhambooking2** [pymc-labs/pymc-marketing#1242] [Q&A]
I have been exploring Pymc TVP MMM modelling. So far i understand that we can incorporate time varying intercept and time varying media coefficient. 

However i see that there is a single time varying media coefficient which is applied across all the media channels. Is there a way to incorporate different time varying coefficient for each media channels indiviudally? Also it would be great to understand the intution behind having a single time varying coefficient across all media channels

**Comment by williambdean** [comment]
At the moment, no way to do that besides with custom model. 
We at PyMC-Labs have found them difficult to converge so the single multiplier was the implementation we went with. 
It is very easy with custom models and hopefully possible via `MMM` class after upcoming few HSGP PRs. See #1250. Keep up with the "TVPs" label to follow or feel free to suggest your use-case in the relevant issues!
